PlutoHelperAgent/AppDelegate.m

```diff
@@ -193,7 +264,13 @@ -    (void)getUrl:(NSAppleEventDescriptor *)event
         BOOL extensionGood = [WhiteListProcessor checkIsInWhitelist:projectPath whiteListName:@"extensions_list" prefix:false];
               
         if (pathGood && extensionGood) {
-            [self tryOpenProject:projectPath];
+            NSArray *pathParts = [projectPath componentsSeparatedByString:@"?"];
+            NSString *partZero = [pathParts objectAtIndex:0];
```

Comment:
what is the function of `partZero`? Please give it a more descriptive name

Reply:
It is also used on line 270.

Reply:
partZero is the zeroth part of the path split on the '?' character.

Reply:
I have changed its name to 'pathPartZero'.